src/maintainer/knowledge_base.rst

```diff
@@ -118,28 +118,55 @@ use a general web search to explore — while these topics are beyond the
 scope of this documentation, there are ample discussions on them on the broader
 Internet.
 
+To bootstrap a conda environment and install ``conda-build``, consider
+`miniforge<https://github.com/conda-forge/miniforge>`__.
```

Comment:
```suggestion
`miniforge <https://github.com/conda-forge/miniforge>`__.
```

Reply:
This was the first "Anonymous hyperlink mismatch" and just pushed another commit that fixes the other. Woops!